test_requirements.txt

```diff
@@ -2,6 +2,7 @@ flake8==3.8.4
 mypy==0.790
 python-dateutil==2.7.5
 django-extensions
-freezegun==0.3.11
+freezegun==0.3.12
 Faker==1.0.1
 django-reversion==3.0.8
+django-germanium
```

Comment:
need this for `test_call_command` function.

Reply:
You could make instance of that command and call it manually, or? Nevermind.